src/adapter/src/catalog/builtin.rs

```diff
@@ -1692,7 +1692,8 @@ pub static MZ_CLUSTER_REPLICA_SIZES: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTa
         .with_column("processes", ScalarType::UInt64.nullable(false))
         .with_column("workers", ScalarType::UInt64.nullable(false))
         .with_column("cpu_nano_cores", ScalarType::UInt64.nullable(false))
-        .with_column("memory_bytes", ScalarType::UInt64.nullable(false)),
+        .with_column("memory_bytes", ScalarType::UInt64.nullable(false))
+        .with_column("disk_bytes", ScalarType::UInt64.nullable(true)),
```

Comment:
Why is this nullable? All sizes have a knowable `disk_bytes` value (which for now is zero since the feature isn't launched), right?

Reply:
@umanwizard Its nullable because I don't want to say ANYTHING about the contents of this column right now

Reply:
OK, fair enough